### ISSUE TITLE
Write Angular tests

### ISSUE THREAD

**Issue by KilianKrause** [reflectoring/coderadar#256]

**Comment by jo2** [comment]
@maximAtanasov did you already start working here? if not i can do so because i have nothing left to do.

**Comment by maximAtanasov** [comment]
@jo2 
No, I've been working on the backend. However, it would be nice if some of the PRs were merged first before we write tests. Currently #270 and #269 (Drafts) both build upon #244 (not merged) and both of those change some things that would impact testing.
